### ISSUE TITLE
Proxy fails for users on 2a cluster

### ISSUE THREAD

**Issue by vpavlin** [fabric8-services/fabric8-jenkins-proxy#150]
Proxy now returns error for all tenants on our second OSO cluster

```
Got status 404 Not Found (404) from https://api.starter-us-east-2.openshift.com/oapi/v1/namespaces/jchevret-osiotest3-jenkins/deploymentconfigs/jenkins
```

This is due to the fact that Idler only knows about one cluster and thus asks wrong cluster for non-existing user

Temporary workaround is to return `false` in `IsIdle` call when a response `StatusCode` is `404`. In other words, if we cannot verify whether the Jenkins is idled, just carry on with actual proxying.

Side effect might be that a user who has never logged into OSIO and does not have any spaces (i.e. does not have OSO namespaces initilased) will get proxied to a non-existent route - which results in "Application Not Available" or some 5xx error - which is the same as if he would access the Jenkins route directly

**Comment by hferentschik** [comment]
Sounds like a good plan. Let's go for that for now. Primarily we want to get Proxy/Idler rolled out for the single cluster case and then tackle the multi-cluster issue. There will be two alternatives we can look into.

**Comment by hferentschik** [comment]
See also issue #151

**Comment by vpavlin** [comment]
This will not work, because Proxy gets route from Idler by calling `GetRoute` (https://github.com/fabric8-services/fabric8-jenkins-proxy/blob/master/internal/proxy/proxy.go#L387) and Idler will fail there as well

**Comment by hferentschik** [comment]
Merged via pull request #154
